Content.Shared/Shuttles/Components/IFFComponent.cs

```diff
@@ -10,11 +10,6 @@ namespace Content.Shared.Shuttles.Components;
 [Access(typeof(SharedShuttleSystem))]
 public sealed partial class IFFComponent : Component
 {
-    /// <summary>
-    /// Should we show IFF by default?
-    /// </summary>
-    public const bool ShowIFFDefault = true;
-
```

Comment:
@metalgearsloth Is this still relevant?

Reply:
I don't think any forks have used it so probably not.